server/command.go

```diff
@@ -197,16 +200,19 @@ func executeConnect(p *Plugin, context *model.CommandArgs, args ...string) *mode
 func executeDisconnect(p *Plugin, commArgs *model.CommandArgs, args ...string) *model.CommandResponse {
 	user, err := store.LoadUser(commArgs.UserId)
 	if err != nil {
-		return p.responsef(commArgs, "Could not complete the **disconnection** request. Error: %v", err)
+		p.client.Log.Error("Error loading  the user from store", "UserID", commArgs.UserId, "error", err.Error())
+		return p.responsef(commArgs, "Failed to complete the **disconnection** request. Error: %v", err)
 	}
 	confluenceURL := user.InstanceURL
 
 	disconnected, err := p.DisconnectUser(confluenceURL, commArgs.UserId)
-	if errors.Cause(err) == store.ErrNotFound {
-		return p.responsef(commArgs, "Your account is not connected to Confluence. Please use `/confluence connect` to connect your account.")
-	}
 	if err != nil {
-		return p.responsef(commArgs, "Could not complete the **disconnection** request. Error: %v", err)
+		if errors.Cause(err) == store.ErrNotFound {
+			return p.responsef(commArgs, "Your account is not connected to Confluence. Please use `/confluence connect` to connect your account.")
```

Comment:
When issuing a disconnect, should an already disconnected account raise an error?  Perhaps, but telling the user to connect their account doesn't seem right to  me.